lib/wavefile-creator.js

```diff
@@ -182,7 +182,7 @@ export class WaveFileCreator extends WaveFileParser {
     }
     if (
       (info.sampleRate == 44100 || info.sampleRate == 22050) &&
-      info.padding
+      !info.padding
```

Comment:
This is the issue that needed to be changed - was checking for if there was padding instead o